src/performance/perfcollect/perfcollect

```diff
@@ -53,548 +53,558 @@ collect_threadTime=0
+    DotNETRuntime:MethodJitInliningSucceeded
+    DotNETRuntime:MethodJitInliningFailed
+    DotNETRuntime:MethodJitTailCallSucceeded
+    DotNETRuntime:MethodJitTailCallFailed
 )
 
 declare -a DotNETRuntime_JittedMethodILToNativeMapKeyword=(
-	DotNETRuntime:MethodILToNativeMap
+    DotNETRuntime:MethodILToNativeMap
 )
 
 declare -a DotNETRuntime_LoaderKeyword=(
-	DotNETRuntime:ModuleDCStartV2
-	DotNETRuntime:ModuleDCEndV2
-	DotNETRuntime:DomainModuleLoad
-	DotNETRuntime:DomainModuleLoad_V1
-	DotNETRuntime:ModuleLoad
-	DotNETRuntime:ModuleUnload
-	DotNETRuntime:AssemblyLoad
-	DotNETRuntime:AssemblyLoad_V1
-	DotNETRuntime:AssemblyUnload
-	DotNETRuntime:AssemblyUnload_V1
-	DotNETRuntime:AppDomainLoad
-	DotNETRuntime:AppDomainLoad_V1
-	DotNETRuntime:AppDomainUnload
-	DotNETRuntime:AppDomainUnload_V1
+    DotNETRuntime:ModuleDCStartV2
+    DotNETRuntime:ModuleDCEndV2
+    DotNETRuntime:DomainModuleLoad
+    DotNETRuntime:DomainModuleLoad_V1
+    DotNETRuntime:ModuleLoad
+    DotNETRuntime:ModuleUnload
+    DotNETRuntime:AssemblyLoad
+    DotNETRuntime:AssemblyLoad_V1
+    DotNETRuntime:AssemblyUnload
+    DotNETRuntime:AssemblyUnload_V1
+    DotNETRuntime:AppDomainLoad
+    DotNETRuntime:AppDomainLoad_V1
+    DotNETRuntime:AppDomainUnload
+    DotNETRuntime:AppDomainUnload_V1
 )
 
 declare -a DotNETRuntime_LoaderKeyword=(
-	DotNETRuntime:ModuleLoad_V1
-	DotNETRuntime:ModuleLoad_V2
-	DotNETRuntime:ModuleUnload_V1
-	DotNETRuntime:ModuleUnload_V2
+    DotNETRuntime:ModuleLoad_V1
+    DotNETRuntime:ModuleLoad_V2
+    DotNETRuntime:ModuleUnload_V1
+    DotNETRuntime:ModuleUnload_V2
 )
 
 declare -a DotNETRuntime_SecurityKeyword=(
-	DotNETRuntime:StrongNameVerificationStart
-	DotNETRuntime:StrongNameVerificationStart_V1
-	DotNETRuntime:StrongNameVerificationStop
-	DotNETRuntime:StrongNameVerificationStop_V1
-	DotNETRuntime:AuthenticodeVerificationStart
-	DotNETRuntime:AuthenticodeVerificationStart_V1
-	DotNETRuntime:AuthenticodeVerificationStop
-	DotNETRuntime:AuthenticodeVerificationStop_V1
+    DotNETRuntime:StrongNameVerificationStart
+    DotNETRuntime:StrongNameVerificationStart_V1
+    DotNETRuntime:StrongNameVerificationStop
+    DotNETRuntime:StrongNameVerificationStop_V1
+    DotNETRuntime:AuthenticodeVerificationStart
+    DotNETRuntime:AuthenticodeVerificationStart_V1
+    DotNETRuntime:AuthenticodeVerificationStop
+    DotNETRuntime:AuthenticodeVerificationStop_V1
 )
 
 declare -a DotNETRuntime_DebuggerKeyword=(
-	DotNETRuntime:DebugIPCEventStart
-	DotNETRuntime:DebugIPCEventEnd
-	DotNETRuntime:DebugExceptionProcessingStart
-	DotNETRuntime:DebugExceptionProcessingEnd
+    DotNETRuntime:DebugIPCEventStart
+    DotNETRuntime:DebugIPCEventEnd
+    DotNETRuntime:DebugExceptionProcessingStart
+    DotNETRuntime:DebugExceptionProcessingEnd
 )
 
 declare -a DotNETRuntime_CodeSymbolsKeyword=(
-	DotNETRuntime:CodeSymbols
+    DotNETRuntime:CodeSymbols
+)
+
+declare -a DotNETRuntime_TieredCompilationKeyword=(
+    DotNETRuntime:TieredCompilationSettings
```

Comment:
I can't remember off the top of my head if these need to be updated or not given that you renamed the keyword from TieredCompilation to Compilation.  Can you check this?

Reply:
Oh right, forgot to update this one, will fix